// try{
//     console.log.aula('Hi, this is error');
// }
try{
    console.log('its error too');
} catch(err){
    console.log('something write here');
    console.log(err);
}